### ISSUE TITLE
SME Committee - Review questions

### ISSUE THREAD

**Issue by JellePraet** [EHDEN/ApplicationPortals#38]
At this moment, you have 1 box to provide a rational for the 4 questions asked (3 scores and interview yes/no). Would it not be better to have 1 box to provide a SHORT rationale per question? I'm afraid that otherwise they will provide a rationale for 1 of the questions but not for all of them.

Also, the **Assignment - do not delete** => I forgot what this stands for? Might need to reword this to be more clear?

And the **Date - do not delete** , is that the date at which we assign the review, or is that the due date of the review?

**Comment by PRijnbeek** [comment]
1. We could indeed. Will put it on the ToDo list

2. That is an internal column we should not delete. These do not show up in the form itself.
3. Same thing.

**Comment by PRijnbeek** [comment]
1. Now say "please provide the rationale for all the scores above"
